src/main/java/ru/sbtqa/tag/pagefactory/PageWrapper.java

```diff
@@ -142,7 +142,7 @@ public Page changeUrlByTitle(String packageName, String title) throws PageInitia
             } else {
                 try {
                     URL currentUrl = new URL(PageFactory.getWebDriver().getCurrentUrl());
```

Comment:
And no, this is not a legacy. When one touches legacy shit and changes  "//" string to a const, he gets involved. So one must make a good code from legacy shit, not a less shitty legacy shit.